pkg/services/control/service.proto

```diff
@@ -345,3 +348,49 @@ message FlushCacheResponse {
     Body body = 1;
     Signature signature = 2;
 }
+
+// ObjectStatus request.
+message ObjectStatusRequest {
+    // Request body structure.
+    message Body {
+        // Requested object.
+        bytes object_address = 1;
```

Comment:
```suggestion
        string object_address = 1;
```

Reply:
hm, we have [this](https://github.com/nspcc-dev/neofs-node/blob/73e84148f8564c5ae71e8b537b6cf0014a96dd47/pkg/services/control/server/gc.go#L32-L42) and [this](https://github.com/nspcc-dev/neofs-node/blob/73e84148f8564c5ae71e8b537b6cf0014a96dd47/pkg/services/control/service.proto#L114-L116), and it confuses me (but i copied it somehow). but agree that in a new command we should use `string`, fixed